src/main/java/com/pru/token/app/RunAfterStartUp.java

```diff
@@ -32,16 +32,16 @@ public void runAfterStartup() {
 		if (roleRepository.findAll().size() < 1) {
```

Comment:
No hard-coded values please, once the application s up, the user needs to set up the users/meta-data/other required data from the application itself. The last demonstration has been done without these values.